ign_rviz_plugins/res/qml/TFDisplay.qml

```diff
@@ -146,6 +146,7 @@ Item {
         text: model.name
         checked: model.checked
         onClicked: {
+          TFDisplay.refresh();
```

Comment:
Because you have removed the `FrameListChanged` even you are updating the GUI combo-box when and where needed. This works great for `GlobalOptions` and `Axes` display plugins but not in TF due to the following reasons:
1. It is not very intuitive on how to refresh the TF list. To view all the frames you have to click on _All Frames_, which refreshes the TreeView but also sets the visibility of all frames to false. You have to click it again to view All Frames.
2. If I launch ign-rviz and load a TF display plugin before running any node that publishes TF data, I can't see the TF visualization until I follow step 1.
3. This refresh is enabled for each and every row of Tree view. This results in clearing the tree list and then repopulating all the frames with a checkbox set to true. Due to this, I can't enable disable visibility of one or more frames individually.

Reply:
Yeah, completely missed that. Have added events for TF display, using a `QTimer` triggered every one second.